services/serializers/api_v3.py

```
@@ -349,6 +352,373 @@
+                title = event.get('title', '')
+                # Prefer html_url, fall back to url
+                url = event.get('html_url') or event.get('url', '')
+                if pr_num and url:
+                    pr_ref = f"pr #{pr_num}"
+                    pr_lookup[pr_ref.lower()] = {'original_text': pr_ref, 'url': url}
+                    if title:
+                        pr_lookup[title.lower()] = {'original_text': title, 'url': url}
+        
+        # Add links to Twitch clips
+        for clip_key, clip_data in clip_lookup.items():
+            original_title = clip_data['original_title']
```

Ruff (0.12.2)

Local variable `original_title` is assigned to but never used

Remove assignment to unused variable `original_title`

(F841)